### DISCUSSION TITLE
oktoberfest how to get started and make quality PRs

### DISCUSSION THREAD

**Discussion by shivambhat** [EddieHubCommunity/support#295] [Q&A]
I have just started  with open source. So here is what I want to know that how can I make quality pull requests for **#oktoberfest**  and  am  I supposed to  submit it  somewhere for oktoberfest?

**Comment by eddiejaoude** [comment]
According to the Oktoberfest website 

> OKTOBERFEST 2020
> THE OKTOBERFEST 2020 WILL NOT TAKE PLACE DUE TO THE CORONA PANDEMIC.

https://www.oktoberfest.de/en
